src/components/Temp/Temp.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {View} from 'react-native';
+import useLocalize from '../../hooks/useLocalize';
+import useWindowDimensions from '../../hooks/useWindowDimensions';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+import variables from '../../styles/variables';
+import BlockingView from '../BlockingViews/BlockingView';
+import * as Illustrations from '../Icon/Illustrations';
+
+const propTypes = {
+    /** Child elements */
+    children: PropTypes.node,
+
+    /** If true, child components are replaced with a blocking "not found" view */
+    shouldShow: PropTypes.bool,
+
+    /** The key in the translations file to use for the title */
+    titleKey: PropTypes.string,
+
+    /** The key in the translations file to use for the subtitle */
+    subtitleKey: PropTypes.string,
+
+    /** The key in the translations files to use for the link */
+    linkKey: PropTypes.string,
+
+    /** Action to run when clicked on a the link text */
+    onLinkPress: PropTypes.func,
+};
+
+const defaultProps = {
+    children: null,
+    shouldShow: false,
+    titleKey: 'mobilePlacerHolder.title',
+    subtitleKey: 'mobilePlacerHolder.subTitle',
+    linkKey: 'mobilePlacerHolder.goBackHome',
+    onLinkPress: () => Navigation.dismissModal(),
+};
+
+function Temp({titleKey, subtitleKey, linkKey, onLinkPress, children, shouldShow}) {
```

Comment:
This could be called WideScreenContent, LargeDisplayWrapper or similar

Reply:
LargeDisplayWrapper This seems promising for me. @hayata-suenaga What's your thought?

Reply:
is this a test component for testing the display of the blocking view or is this The blocking view?